### PR TITLE
Removes mulligan from Blob and Wizard

### PR DESCRIPTION
**Pull request by Robustin**
My only regret was not doing this sooner. I got talked out of it last year with some "alternative fix" that never ended up working. 

There is absolutely no reason to punish the crew for doing exactly what we expect them to do during a solo antag mode, team up, cooperate, empower one another... just so people can get shot in the back with the gun they had just given away. 

It doesn't shake up the meta, at this point if the round doesnt end after blobs/wizards everyone knows all hell is about to break loose with random traitors and lings. 

I didn't touch the summon events exception since summon events is already "Cancer: The Mode" I wouldn't want to ruin the theming. 

[This image sums up what happens every mulligan](http://imgur.com/a/InrEN)

Post-Round Edit: [This image sums up how every mulligan ends](http://imgur.com/a/CvCM9)

### CODE REVIEW

**Comment by coiax** [comment]
Needs a changelog. Also when @lzimann finishes datum antags, it'll all be good.

**Comment by vcordie** [comment]
That round aint over yet you cuck. But yeah, this is a good change and absolutely the right thing to do, at least for blob.

**Comment by Magikarpet** [comment]
I agree with blob not needing a force mulligan without admin intervention, but I heavily disagree on the wizard note. Wizards generally won't forcemulligan unless it's highpop and the wizard kills themselves instantly. While that may be hilarious, even summon guns/summon magic adds something interesting.

**Comment by ExcessiveUseOfCobblestone** [comment]
summonguns and magic spawns traitors anyways though...?

**Comment by Magikarpet** [comment]
Yeah, which makes it somewhat hilarious when the entire crew is armed and a traitor immediately gets gunned down as a consequence of it. Even in summon guns/magic there is some counterplay for both loyalst and survivors to survive, and mulligan adds even more chaos to an already chaotic roundtype.

**Comment by lzimann** [comment]
Summon guns/magic creates traitors which should override it ending when the wiz dies(not 100% sure since I haven't read the mulligan part of the code). 
Plus we have configs for this, I'll actually turn the continuos off for these modes since I agree that them having mulligan is kinda stupid.

**Comment by Robustin** [comment]
To be honest im not 100% sure either, and I'm reallllly iffy on summon guns since many wizards will legitimately cast that for the free point on the gambit that everyone will kill each other and they can learn one more spell. 

Summon magic is kind of intermediate since a wizard doesn't immediately benefit from casting it but can definitely rack up a ton of artifacts fast. 

Summon events is just pure memes while I hate it from the bottom of my heart, it would seem unfair for the wizard to spam events only to have it all end 10 seconds later when they get mass bluespaced into the engine or something. 

If its not the case that the first two spells don't already prevent the round from ending, it wouldn't be hard to use the same exception that summon events has.

**Comment by PKPenguin321** [comment]
Summon guns/magic doesn't give the wizard a point any more, it costs one

**Comment by Cheridan** [comment]
👍 for blob, 👎 for wizard. mulligan was basically MADE to compensate for fail wizards dying in 2 minutes and forcing the entire server to do the 10 minute roundstart routine all over again

**Comment by Aloraydrel** [comment]
blobs already dead tbh

**Comment by Robustin** [comment]
Roundstart routine should be 4 minutes at most. As soon as extra access/weapons go out for the wizard is the moment I lose interest in having the round mulligan'd. I'm sure some wizards might still cuck themselves instantly but its really not an issue now they all smash every summon button at roundstart (if theyre bad) or survive sufficiently long anyway (if theyre good).

**Comment by optimumtact** [comment]
Is blob even enabled? I had assumed it was a dead gamemode waiting for someoen to remove or improve?

**Comment by Incoming5643** [comment]
You are doing this all wrong, if you don't want those modes to mulligan there are config options for that. Change the default config options, don't just remove optional functionality!

File: code/game/gamemodes/wizard/wizard.dm [hunk 1]
```diff
@@ -166,7 +166,7 @@
 
 	for(var/datum/mind/wizard in wizards)
 		if(isliving(wizard.current) && wizard.current.stat!=DEAD)
-			return ..()
+			return 1
```
Comment (Incoming5643) [inline code/game/gamemodes/wizard/wizard.dm]: Also this isn't even the correct thing to do. This for loop checks for other living wizards, so you've just told the game to end every wizard round as soon as it starts!

**Comment by optimumtact** [comment]
and yeah this is a server config

**Comment by Steelpoint** [comment]
I support this change, defeating a Wizard is a monumental task in of itself for the crew, I always expect the round to end shortly after the Wizard is defeated yet when the round continues I tend to just bolt myself in my office to avoid the shitshow that is about to begin.

**Comment by optimumtact** [comment]
literally a server configuration issue, this does not need to be hardcoded in code

**Comment by KorPhaeron** [comment]
Convince he admins to change config

**Pull request closed without merging**